fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -155,80 +147,86 @@ def format_df(df: pd.DataFrame) -> pd.DataFrame:
     return df
 
 
-def query_categories(category_keywords: Dict[str, List[str]]) -> pd.DataFrame:
+def create_report1(
+    df: pd.DataFrame, category_keywords: Dict[str, List[str]] = KEYWORDS
+) -> (pd.DataFrame, pd.DataFrame):
+    """Report 1: counts and latest/oldest message timestamps"""
+    # No-results report
+    reports: List[Dict] = []
+    no_result_keywords: List[str] = []
+    for category, keywords in category_keywords.items():
+        for k in keywords:
+            df_kw = df[df['keyword'] == k]
+            df_kw = df_kw.sort_values(['timestamp'])  # oldest first
+            kw_report = {
+                'category': category,
+                'keyword': k,
+                # TODO: Check: If a message contains the keyword more than once, will it return more than 1 result? or
```

Comment:
@rohaher @DaveraGabriel # TODO: Check: If a message contains the keyword more than once, will it return more than 1 result? or simply 1 message result?

Will bring this up at the meeting.